cabal-install/src/Distribution/Client/ProjectConfig/Legacy.hs

```diff
@@ -256,38 +260,66 @@ parseProject
   -> ProjectConfigToParse
   -- ^ The contents of the file to parse
   -> IO (ProjectParseResult ProjectConfigSkeleton)
-parseProject rootPath cacheDir httpTransport verbosity configToParse =
-  do
-    let (dir, projectFileName) = splitFileName rootPath
-    projectDir <- makeAbsolute dir
-    projectPath <- canonicalizeConfigPath projectDir (ProjectConfigPath $ projectFileName :| [])
-    parseProjectSkeleton cacheDir httpTransport verbosity projectDir projectPath configToParse
-    -- NOTE: Reverse the warnings so they are in line number order.
-    <&> \case ProjectParseOk ws x -> ProjectParseOk (reverse ws) x; x -> x
+parseProject rootPath cacheDir httpTransport verbosity configToParse = do
+  let (dir, projectFileName) = splitFileName rootPath
```

Comment:
It'd be preferable to not change layout/identation without a real need, I think.

Reply:
I strive for minimal diffs. In this situation, I was rebasing something older and it made the merge conflict resolution easier this way.

https://github.com/haskell/cabal/blame/51e1817dec9980d0d204a051fc3c4dc981c3d027/cabal-install/src/Distribution/Client/ProjectConfig/Legacy.hs#L259-L266

https://github.com/haskell/cabal/blame/54d364df92b654ce712f0fe277ed709e710d8bf4/cabal-install/src/Distribution/Client/ProjectConfig/Legacy.hs#L246-L250

I actually prefer the way it was before the latest commit, with the `= do` on the same line rather than `do` by itself on its own line. This also has less indentation.

If you're OK with the way it is, then I'd like to keep it that way, the way it was, one commit before the last commit.

Reply:
I'm a little surprised that fourmolu doesn't always normalize.

Reply:
Or it could have been I used a more recent version of fourmolu at some stage.